src/components/message-input/index.tsx

```diff
@@ -357,7 +353,7 @@ export class MessageInput extends React.Component<Properties, State> {
   }
 
   get allowFileAttachment() {
-    return !featureFlags.enableMatrix && !this.props.isEditing;
+    return !this.props.isEditing;
   }
```

Comment:
Ok, but enabling this will allow a user to select / drag any type of file, right? As in, do we need more checks to prevent other files?

Reply:
yes, it would. That request will just fail because we only support sending the images for now. I left it thinking files are gonna be a recent follow up. I can look into adding more checks, to only allow images to be "sent" via the UI for now. Let me know

Reply:
Did you see how it fails? What does the user experience? If they're getting ok feedback then it's fine but we can't just let things get into bad states.

Reply:
This is how it fails. I think that's an okay feedback 🤔 
<img width="715" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/f5999242-2634-4a2b-84f0-fe4cd170a77f">

Reply:
How does the user know why that failed? If I were a user I'd just keep trying.

Reply:
so... it was actualy pretty simple. Just needed to comment these out. 😅 
<img width="459" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/82292b2a-fc03-4783-b79d-fe610810780f">

added a note. We're only allowing images to be attached for now.